src/otx/v2/adapters/torch/mmengine/mmpretrain/modules/datasets/otx_datasets.py

```diff
@@ -72,39 +74,35 @@ def get_indices(self, num_classes: list, *args) -> dict:
     def load_annotations(self) -> None:
         """Load annotations."""
         include_empty = self.empty_label in self.labels
-        for i, _ in enumerate(self.otx_dataset):
+        for item in self.otx_dataset:
+            item_labels = [ann.label for ann in item.annotations]
```

Comment:
again, a bit opinionated, you don't need to accept :)
```suggestion
            item_labels = [annotation.label for annotation in item.annotations]
```

Reply:
updated at 0e4de99acf5e9b1e54d3375cba358c4c02d964b2.